<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToSrtDialog</class>
 <widget class="QDialog" name="ToSrtDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Seleccionar subtitulos a descomprimir</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="acceptCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="subtitlePath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="subtitleFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Archivo de subtitulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="copyFiles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Copiar al descomprimir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="copyPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="copyRoute">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ruta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OCR A Extended</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>acceptCancel</sender>
   <signal>accepted()</signal>
   <receiver>ToSrtDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptCancel</sender>
   <signal>rejected()</signal>
   <receiver>ToSrtDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
